example/models.py

```diff
@@ -34,3 +37,17 @@ def import_wordpress_data(self, data):
 
         # own model fields
         self.body = data["body"]
+
+
+@register_snippet
+class Category(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
**Suggestion**: We're using this as a "get_or_create" field. It would be good to add a uniqueness constraint.